dtfs-central-api/src/v1/routes/tfm-routes.js

```diff
@@ -722,4 +724,33 @@ tfmRouter.route('/users/team/:teamId')
     tfmUsersController.findTfmTeamUser,
   );
 
+/**
+ * @openapi
+ * /tfm/utilisation-reports/set-status:
+ *   put:
+ *     summary: Put utilisation report status for multiple utilisation reports
+ *     tags: [TFM]
+ *     description: Set the status of many utilisation reports to completed or not completed.
+ *     requestBody:
+ *       content:
+ *         application/json:
+ *           schema:
+ *             type: array
+ *             items:
+ *               oneOf:
+ *                 - $ref: '#/definitions/ReportStatusWithReportId'
+ *                 - $ref: '#/definitions/ReportStatusWithBankId'
+ *           required: true
+ *     responses:
+ *       204:
+ *         description: OK
+ *       400:
+ *         description: Bad request
+ */
+tfmRouter
+  .route('/utilisation-reports/set-status')
+  .put(
+    tfmPutUtilisationReportStatusController.putUtilisationReportStatus,
+  );
```

Comment:
I believe this should be specific to `tfm` as there isn't a non-`tfm` case where this endpoint should be accessed (that was my logic behind it, anyway). With respect to the other `utilisation-reports` routes, these can be accessed by both `portal` and `tfm` so they should be left as general (such as downloading the report, for instance)

Reply:
My feeling on this is that the central API should be general, so shouldn't necessarily know what it is being used for. Since utilisation reports are not TFM specific then the route should be too. I realise though that that may not be the logic that has been used consistently in the project though.

@ChristianMcc - would be good to get your thoughts on this one.

Reply:
My feeling on this is that the central API should be general, so shouldn't necessarily know what it is being used for. Since utilisation reports are not TFM specific then the route should be too. I realise though that that may not be the logic that has been used consistently in the project though.

@ChristianMcc - would be good to get your thoughts on this one.

Reply:
I'd go with the utilisation-reports route partly just because it keeps all this logic together in one file and I think that's quite useful when it's a contained feature. And it lines up with me agreeing that central API should be more general than catering exactly to it's consumers

Reply:
Sounds good to me then